### DISCUSSION TITLE
How to override Tailwind Typography with tailwind classes?

### DISCUSSION THREAD

**Discussion by ivangretsky** [tailwindlabs/tailwindcss#15485] [Help]
Good day everyone!

I am looking for a way to customize prose styles and align their styling with regular element styles. For example I have `h1` tags styled like this with tailwind classes:

```html
<h1 class="mb-6 font-normal text-2xl lg:text-3xl">
    Please help me with this!
</h1>
```
All the classes are abstracted in a php variable so I can change them consistently. 

Now I need to set `h1` in **prose** so it looks exactly like `h1` with tailwind classes above. It is possible in `tailwind.config.js` like explained [here](https://github.com/tailwindlabs/tailwindcss/discussions/9001). But it is rather cumbersome to keep them in sync.

I dream to find a way to style elements in **prose** with the same classes. Is there a way to do it? Maybe I could use something like @apply in `tailwind.config.js`? Any other suggestions?

**Comment by asimpletune** [comment]
In the h1's parent container you can do something like `prose-h1:font-normal`
